engine/server/engine/centralized_logs/client_implementations/persistent_volume/log_remover/log_remover_test.go

```diff
@@ -0,0 +1,68 @@
+package log_remover
+
+import (
+	"fmt"
+	"github.com/kurtosis-tech/kurtosis/engine/server/engine/centralized_logs/client_implementations/persistent_volume/logs_clock"
+	"github.com/kurtosis-tech/kurtosis/engine/server/engine/centralized_logs/client_implementations/persistent_volume/volume_consts"
+	"github.com/kurtosis-tech/kurtosis/engine/server/engine/centralized_logs/client_implementations/persistent_volume/volume_filesystem"
+	"github.com/stretchr/testify/require"
+	"strconv"
+	"testing"
+	"testing/fstest"
+)
+
+const (
+	logsStorageDirpathForTests = "var/log/kurtosis/"
+
+	testEnclaveUuid      = "test-enclave"
+	testUserService1Uuid = "test-user-service-1"
+
+	defaultYear = 2023
+	defaultDay  = 0
+)
+
+func TestLogRemover_Run(t *testing.T) {
+	week49filepath := getWeekFilepathStr(49)
+	week50filepath := getWeekFilepathStr(50)
+	week51filepath := getWeekFilepathStr(51)
+	week52filepath := getWeekFilepathStr(52)
+	week1filepath := getWeekFilepathStr(1)
+	week2filepath := getWeekFilepathStr(2)
+
+	mapFs := &fstest.MapFS{
+		week49filepath: {
+			Data: []byte{},
+		},
+		week50filepath: {
+			Data: []byte{},
+		},
+		week51filepath: {
+			Data: []byte{},
+		},
+		week52filepath: {
+			Data: []byte{},
+		},
+		week1filepath: {
+			Data: []byte{},
+		},
+		week2filepath: {
+			Data: []byte{},
+		},
+	}
+
+	mockFs := volume_filesystem.NewMockedVolumeFilesystem(mapFs)
+	currentWeek := 2
+
+	mockTime := logs_clock.NewMockLogsClock(defaultYear, currentWeek, defaultDay)
+	logRemover := NewLogRemover(mockFs, mockTime)
+
+	// log remover should remove week 49 logs
+	logRemover.Run()
+
+	_, err := mockFs.Stat(week49filepath)
+	require.Error(t, err) // check the file doesn't exist
+}
+
+func getWeekFilepathStr(week int) string {
+	return fmt.Sprintf("%s%s/%s/%s%s", logsStorageDirpathForTests, strconv.Itoa(week), testEnclaveUuid, testUserService1Uuid, volume_consts.Filetype)
```

Comment:
Same here like I'm a bit confused, is that really the path that will get produced by vector, even with the `%%Y` you just added? Where is the year here?

Reply:
No - added the year.